### PR TITLE
Overhauls Observer UI

### PR DESCRIPTION
**Pull request by Tsar-Salat**
Ports:
- https://github.com/tgstation/tgstation/pull/57471
- https://github.com/tgstation/tgstation/pull/61638

## About The Pull Request
Adds spawner and Minigames menu from /tg/

Removes "Jump to Mob" Button

## Why It's Good For The Game

The Spawners menu and CTF are both existing features that players can use, but its not obvious to new players because the former is hidden deep in the statpanel, and the latter requires you to know you can jump to it and click on it to request a CTF game to start.

Ghostroles/Minigames are a good way to practice game mechanics, especially for new players. It is a prerogative to make them more accessible.

I removed the Jump To Mob Button as it became redundant when we Converted the Orbit button to TGUI.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://github.com/BeeStation/BeeStation-Hornet/assets/62388554/855cbdc1-5b61-4a8c-887d-c573f30a84e6

</details>

## Changelog
:cl: rkz, NamelessFairy, Mothblocks
add: added spawners button to observer UI
add: added Minigames button to observer UI
remove: removed Jump To Mob button from Observer UI. The Orbit button already lists all mobs and is the superior tool anyway. Admins can use Jump to Key anyway.
rscadd: minigame, spawner icons
/:cl:

### CODE REVIEW

**Comment by Tsar-Salat** [comment]
IDIOT

image

STUPID MORON

**Comment by Tsar-Salat** [comment]
ohhh my fucking gooooood

File: tgui/packages/tgui/interfaces/CTFPanel.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { useBackend } from '../backend';
+import { Box, Button, Section, Flex, Stack, Divider } from '../components';
+import { Window } from '../layouts';
+
+export const CTFPanel = (props, context) => {
+  const { act, data } = useBackend(context);
+  const teams = data.teams || [];
+  const enabled = data.enabled || [];
+  return (
+    <Window title="CTF Panel" width={700} height={600}>
```
Comment (itsmeow) [inline tgui/packages/tgui/interfaces/CTFPanel.js:10]:
```suggestion
    <Window title="CTF Panel" width={700} height={600} theme="generic">
```

**Review by itsmeow — CHANGES_REQUESTED** [review]
We don't have enough ghost minigames to warrant a whole button and menu for it. My suggestion is adding a tab to the spawners menu and removing the minigames button, or merge spawners into the minigames button.

**Comment by itsmeow** [comment]
Also remove the `mafia` icon that was added.

**Comment by Tsar-Salat** [comment]
> We don't have enough ghost minigames to warrant a whole button and menu for i

Adding a button front and center so new players can see it and know CTF exists is a dealbreaker for me.

**Pull request closed without merging**